editor.planx.uk/src/@planx/components/PlanningConstraints/Public.tsx

```diff
@@ -86,11 +86,7 @@ function Component(props: Props) {
   const { data, mutate, isValidating } = useSWR(
     () => (x && y && latitude && longitude ? teamGisEndpoint : null),
     fetcher,
-    {
-      shouldRetryOnError: true,
-      errorRetryInterval: 500,
-      errorRetryCount: 1,
-    }
```

Comment:
I opted to remove these retries since we're using `mutate` here & the applicant will be presented with the option to manually "Search again" in case of an error, so makes sense to me to fail faster! 
![Screenshot from 2023-06-09 10-01-03](https://github.com/theopensystemslab/planx-new/assets/5132349/a1038a05-9b6b-4294-a2ec-59df0d4c6e1e)